shared/common-adapters/animation.native.tsx

```diff
@@ -13,6 +13,7 @@ class Animation extends React.Component<Props> {
           autoPlay={true}
           loop={true}
           source={animationData[this.props.animationType]}
+          // @ts-ignore TODO fix styles
```

Comment:
the styles we have for native are pre-TS and don't match. i'll ticket fixing that eventually